http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppRoutes.scala

```diff
@@ -247,7 +247,7 @@ object AppRoutes {
       "labels"
     )(x =>
       (x.cloudContext,
-       x.cloudContext,
+       x.cloudContext.asString,
```

Comment:
it's a class that has `asString`....when it's CloudContext.Gcp, this is equivelant of the previous value